compiler-plugin/src/main/java/io/ballerina/stdlib/http/compiler/HttpServiceValidator.java

```diff
@@ -52,6 +65,7 @@
     @Override
     public void perform(SyntaxNodeAnalysisContext syntaxNodeAnalysisContext) {
         ServiceDeclarationNode serviceDeclarationNode = (ServiceDeclarationNode) syntaxNodeAnalysisContext.node();
+        extractServiceAnnotationAndValidate(syntaxNodeAnalysisContext, serviceDeclarationNode);
```

Comment:
Move this to L77 as we do some listener validation from L70 to L76. This check should be done afterwards